pkg/ocm/clusters.go

```diff
@@ -33,6 +33,8 @@ import (
 	errors "github.com/zgalor/weberr"
 )
 
+const DoubleQuotesToRemove = "\"\""
```

Comment:
@vkareh not sure about this term. Maybe you have something more clear....

Reply:
no, just remove this and have the user actually delete the contents of the field:

if the cluster has data in httpProxy, the prompt will be populated:
- if they leave the populated data in place, we don't modify it
- if they change the populated data, we update the cluster
- if they delete the populated data, we delete it from the cluster

if the cluster does not have data, the prompt will show up empty:
- if they leave the prompt empty, we don't modify it
- if they enter data, we send it to the cluster

What other options do you see as necessary here?

Reply:
@vkareh I still do not understand how the user deletes the existing value. The prompt is like:
```
? HTTP proxy: [? for help] (http://aa.com)
```
You cannot delete it. Pressing backspace does not delete the value as it's part of the question.